scripts/provisioning/configcmd.py

```diff
@@ -57,6 +58,10 @@ def process(self, configure_only_openldap = False, configure_only_haproxy = Fals
     self.logger.info("validations completed")
 
     try:
+      self.logger.info("update motr max units per request started")
+      self.update_motr_max_units_per_request()
```

Comment:
Add basic validation

more than 1 and less than 128

Reply:
fixed